mods/lord/World/sunshine/src/sunshine.lua

```diff
@@ -0,0 +1,27 @@
+require('commands')
+
+local lighting_default = {
+	shadows = { intensity = 0.33},
+	bloom = { intensity = 0.05 },
+	volumetric_light = { strength = 0.2 },
+}
+
+
+if minetest.settings:get_bool("enable_weather") == false then
+
+	return
+end
+
+function user_light_set(player, value)
```

Comment:
для геттеров и сеттеров префиксы `get`/`set` вначале названия ф-ции.
функция не должна быть глобальной.
```suggestion
local function set_user_light(player, value)
```

Reply:
это как раз часть внутреннего api (только для самого мода, не отдавая его наружу), которое стоит держать в отдельном файле, как у тебя и было и подключать его и тут, и в `commans.lua`